input/projects/why-join-wip.md

```diff
@@ -0,0 +1,51 @@
+# Why should my project join the .NET Foundation?
+
+The .NET Foundation offers administrative and technical services to open source projects to help maintainers govern and develop their projects.
+
+.NET Foundation services aim to increase project stability and maturity and don't replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.
+
+Project benefits include:
+
+## Project guidance and mentoring
+
+New projects joining us will get mentorship on how to best run an open source community along with access to developers currently seeking .NET projects. We work with project leads to help ensure the project grows into a vibrant and welcoming community. And, of course, we love to share best practices for managing a friendly, collaborative, and innovative open source workplace, wherever you may be.
```

Comment:
**QUESTION:** Could you give examples of mentoring and a vibrant community?

Reply:
The mentoring has come during the maintainer's committee meetings as something they'd like to adopt in the near future.

Reply:
I can't give examples, I'm just playing around with content.

I think some more detail from people who know more about this area would be useful.

Reply:
@JamesNK it's being discussed as something we'd like to start adopting in the maintainer's committee but not something currently that's provided beyond the initial onboarding help with the infrastructure.